lxd/instance/drivers/driver_qemu.go

```diff
@@ -5157,3 +5157,52 @@ func (vm *qemu) writeInstanceData() error {
 
 	return nil
 }
+
+// Returns "qemu" and the currently loaded qemu version, 0 if not found.
+// Unsure if this is the proper behavior for a not found, but the logic is there.
+// TODO: This seems to be getting a 0 value for vm.architecture, and as a result cannot find the binary, giving a 0 for the version no.
+func (vm *qemu) Info() instance.Info {
+
+	var qemuVersion, qemuBinary, qemuPath string
+	var out []byte
+	var err error
+	qemuBinary, _, err = vm.qemuArchConfig()
+	println("vm architecture ", vm.architecture)
```

Comment:
@wester162 for setting `vm.architecture` I think it would be worth using `osarch.ArchitectureGetLocalID()` if its not already set, this way it would inherit the systems architecture.

Reply:
Using that to set ``vm.architecture`` works like a charm, and I've worked it into the most recent commit.